spec.bs

```diff
@@ -775,6 +772,16 @@ The user agent MAY utilize the information in
 into a language and using locale-based formatting consistent with that of the
 website.
 
+The user agent MAY decide to skip steps 2 and 3 of the
+{{PaymentRequest/show|PaymentRequest.show()}} method, i.e. not require a user
+activation, when the [=Secure Payment Confirmation payment handler=] is
+selected.
+
+NOTE: The user agent may still decide to enforce the user activation
+requirement, or implement a relaxed requirement such as allowing one
+activationless call per page load. The user agent may also consider other spam
+mitigations such as an anti-clickjacking mechanic.
```

Comment:
I'd like to ask for some clarification regarding the last sentence. The main point of this language is that the UA can skip steps 2 and 3. The text then says that the UA can still do 2 and 3, or do 2 and 3 once per page load. The relationship of the final sentence to steps 2 and 3 is not clear to me. Could it be made clearer (or moved somewhere else)? 

One idea is to add some bits to the security considerations section that talk about click-jacking. See this pull request (intended for the explainer) for inspiration:
https://github.com/w3c/secure-payment-confirmation/pull/230/files

Reply:
+1 to adding an entry to the Security Considerations section. Probably a new 10.3, "Lack of user activation requirement", which should talk to the possible risks (spam + click-jacking) and how a user agent can mitigate them (choose to require an activation, only allow one free activation, and the user-agent may also do anti-click jacking mechanisms such as ignoring clicks immediately after the dialog is shown).

There is also https://github.com/w3c/secure-payment-confirmation/pull/238, which this new Security Considerations sub-section can refer to as well once I land it, to note that a background tab cannot silently trigger SPC.

Reply:
Done